external_install.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Author: Stanislav Laznicka <slaznick@redhat.com>
+# Description:
+#       This simple bash script serves to simplify testing of FreeIPA
+#       installations with an external CA
+
+DBDIR="/abs/path/to/nssdb" # will be removed if exists!
+PWDFILE="$DBDIR/pwdfile.txt"
+NOISE="$DBDIR/noise.txt"
+PASSWORD="milan_je_sulin123"
```

Comment:
Can we please use the commonly used `Secret123` or rather `Secret.123` (for nss fips compliance)?